test/clj/rems/test_browser.clj

```diff
@@ -883,11 +883,27 @@
              (slurp-fields :form)))
       (testing "preview"
         ;; the text is split into multiple DOM nodes so we need btu/has-text?, :fn/has-text is simpler for some reason
-        (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
-                           "(max 127 characters)"))
-        (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
-                           "(optional)"))
-        (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))))
+        (btu/wait-visible {:tag :button :fn/has-class :info-button})
+        (let [button (first (btu/query-all {:tag :button :fn/has-class :info-button}))]
+          (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
+                             "(max 127 characters)"))
+          (is (btu/has-text? {:tag :label :class :application-field-label :fn/has-text "Text area (EN)"}
+                             "(optional)"))
+          (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))
+          (is (not (btu/visible? {:tag :div :fn/has-class :info-collapse})))
+          (is (not (btu/visible? {:tag :div :fn/has-text "Info text (EN)"})))
+          (btu/click-el button)
+          (btu/wait-visible {:tag :div :fn/has-class :info-collapse})
+          (is (btu/visible? {:tag :div :fn/has-text "Info text (EN)"}))
+          (btu/click-el button)
+          (not (is (btu/visible? {:tag :div :fn/has-class :info-collapse})))
+          (change-language :fi)
+          (btu/wait-visible {:tag :label :class :application-field-label :fn/has-text "Text area (FI)"})
+          (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Text area (FI)"}))
+          (btu/click-el button)
+          (btu/wait-visible {:tag :div :fn/has-class :info-collapse})
+          (is (btu/visible? {:tag :div :fn/has-text "Info text (FI)"})))
+        (change-language :en)))
```

Comment:
I think this looks odd with this line hanging here and this should be in the same block as above `change-language` statements. Also if you would check the language change while the field is still open you could remove extra click of the button.